testing/test-run.js

```diff
@@ -75,7 +75,7 @@
       });
 
       afterEach('save output', async function() {
-        let time = moment.unix(this.currentTest.value.timestamp).utcOffset(0).format()
+        let time = moment.unix(this.currentTest.value.timestamp).utcOffset(0).format('YYYY-MM-DD_HH-mm-ss');
```

Comment:
I changed this as this was trying to write the output in a filename that contains ':' character which are invalid on Windows.